developer-docs/bolt-server.md

```diff
@@ -174,7 +174,64 @@ For example:
 If the task runs the response will have status 200.
 The response will be a standard bolt Result JSON object.
 
-## Install from Source
+## Running bolt-server
+
+## Running from source
+
+From your checkout of bolt run
+
+```
+BOLT_SERVER_CONF=config/local.conf bundle exec puma -C puppet_config.rb
+```
+
+
+setup your environment for running commands with
+
+```
+export BOLT_CACERT=spec/fixtures/ssl/ca.pem
+export BOLT_CERT=spec/fixtures/ssl/cert.pem
+export BOLT_KEY=spec/fixtures/ssl/key.pem
+export BOLT_ROOT=https://localhost:62658
+```
+
+you can now make a curl request to bolt which should have an empty response
+
+```
+curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
+```
+
+## Running in a container
+
+from your checkout of bolt build and run the container with
+
+```
+docker run --rm -p 62658:62658 `docker build -q ./`
+```
+
+```
+export BOLT_CACERT=spec/fixtures/ssl/ca.pem
+export BOLT_CERT=spec/fixtures/ssl/cert.pem
+export BOLT_KEY=spec/fixtures/ssl/key.pem
+export BOLT_ROOT=https://localhost:62658
+```
+
+setup your environment for running commands with
+
+```
+export BOLT_CACERT=spec/fixtures/ssl/ca.pem
+export BOLT_CERT=spec/fixtures/ssl/cert.pem
+export BOLT_KEY=spec/fixtures/ssl/key.pem
+export BOLT_ROOT=https://localhost:62658
+```
+
+you can now make a curl request to bolt which should have an empty response
+
+```
+curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
+```
+
+## Run from a Package on A VM
```

Comment:
I don't think this section is particulairly useful anymore. If you want a vm you probably want the rest of PE as well. Should I delete it?

Reply:
Or include a link to the PE docs for setting up a test environment.